docs/faq.md

```diff
@@ -42,6 +42,27 @@ https://docs.docker.com/desktop/
 **A:**
 -->
 
+<!-- #640 -->
+**Q: How can I restart Rancher Desktop?**
+
+**A:** Click **File** and then **Exit**, or right click on the tray icon at the bottom of the screen and choose to quit the app there.
+
+**Q: How can I perform a clean uninstall of Rancher Desktop?**
+
+**A:** You have two options for this procedure:
+
+1. Click on the **Troubleshooting** tab in the Rancher Desktop UI. Once there, click on **Factory Reset**: this will reset to defaults and remove all user data such as Kubernetes, any caches, any configuration, logs, etc. After completing this step, Rancher Desktop should restart and start fresh.
+
+1. Click on **Add or Remove Programs** in your system settings, then uninstall Rancher Desktop there.
```

Comment:
uninstall process is different for different OSs and the steps are covered under the Installation section. we just link to those sections from here?

Reply:
Removed per feedback and instead linked to installation page.